samples/batch/python/README.md

```diff
@@ -1 +1,15 @@
 Examples in C# and Python on how to use the Speech Services Batch Transcription API.
```

Comment:
Why is C# mentioned here?

Reply:
(And maybe "Example" instead of "Examples"?

Reply:
correct, changing it.